pkg/function/kube_exec.go

```diff
@@ -91,15 +91,11 @@ func (kef *kubeExecFunc) Exec(ctx context.Context, tp param.TemplateParams, args
 	if err = Arg(args, KubeExecCommandArg, &cmd); err != nil {
 		return nil, err
 	}
-	stdout, stderr, err := kube.Exec(cli, namespace, pod, container, cmd, nil)
-	format.LogWithCtx(ctx, pod, container, stdout)
-	format.LogWithCtx(ctx, pod, container, stderr)
-	if err != nil {
+	if err := kube.ExecOutput(cli, namespace, pod, container, cmd, nil, os.Stdout, os.Stderr); err != nil {
 		return nil, err
 	}
 
-	out, err := parseLogAndCreateOutput(stdout)
```

Comment:
I think this is still required if used by the ActionSet to set output artifacts

Reply:
Good catch; will fix.

Reply:
Fixed in 17c673ff35f024b5757c22b2894002278bde4839. I used this blueprint to emulate a phase output:

```sh
apiVersion: cr.kanister.io/v1alpha1
kind: Blueprint
metadata:
  name: timer
  namespace: kanister
actions:
  tail:
    outputArtifacts:
      output:
        keyValue:
          foo: "{{ .Phases.tail.Output.foo }}"
    phases:
    - func: KubeExec
      name: tail
      args:
        namespace: default
        pod: timer
        container: timer
        command:
        - sh
        - "-c"
        - |
         /bin/tail_datetime.sh
         echo '###Phase-output###: {"key":"foo","value":"bar"}'
```